_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,299 @@
+    [Electrum Personal Server][eps], which gives particular weight to
+    the conclusion of his email: "I intend to create this CoinSwap
+    software.  It will be almost completely decentralized and available
+    for all to use for free."
+
+- **New node-to-wallet middleware:** Nadav Ivgi [announced][ivgi bwt]
+  the alpha release of Bitcoin Wallet Tracker (BWT), a program that
+  interacts with Bitcoin Core's wallet using its standard RPC interface,
+  uses that data to build additional indexes necessary for lightweight
+  wallets, and then makes that data available via both the Electrum
+  Server protocol and BWT's own [extensive HTTP-based API][bwt api].
+  Similar to Electrum Personal Server, this allows users who prefer the
+  UI of a lightweight wallet (such as Electrum) to retrieve block and
+  transaction data from their own full verification node for additional
+  security.  There's no significant overhead to BWT's approach: its
+  additional indexes are stored only in memory and it can work with
+  pruned nodes in many cases, allowing a combined `bitcoind` and `bwt`
+  setup to use only a few gigabytes of disk space.
+
+    Ivgi also provides a [plugin][bwt plugin] that simplifies setting up
+    BWT with an Electrum client, and it may also be possible to use BWT
+    with other wallets that support the Electrum Server protocol, such
+    as [Edge][], [Blue Wallet][], [Eclair mobile][], and [Phoenix][].
+
+    BWT's HTTP protocol supports additional features beyond those
+    available in the Electrum Server protocol, such as key origin
+    information useful for interaction with HD wallets and wallet
+    collaboration tools such as [PSBT][topic psbt].  His email also
+    notes that future versions of BWT may support [output script
+    descriptors][topic descriptors], allowing wallets to produce and
+    consume standardized descriptions of their script templates.
+
+- **Transaction size calculators:** Jameson Lopp [posted][lopp size] to
+  the Bitcoin-Dev mailing list with links to a [transaction size
+  calculator][lopp calc] he'd developed as well as a similar [calculator
+  developed by Optech][optech calc].  Neither tool claims to be complete
+  or bug-free, but both should be useful to developers who want to
+  quickly compare the sizes of different types of transactions.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **LN backups:** Christian Decker presented at Potzblitz on the latest
+  state of LN backups. He discussed the approaches of other
+  implementations such as Eclair and LND before explaining why
+  C-Lightning is using a synchronous database log plugin. Later, he
+  described why LN backups are more complex than onchain backups, the
+  prospects of getting [SIGHASH_NOINPUT][topic sighash_noinput] or
```

Comment:
maybe prospects of s/getting XYZ into Bitcoin/having XYZ merged into Bitcoin/

Reply:
@michaelfolkson actually used "merge [...] into Bitcoin Core" in his original text, but I changed it to the current phrasing because I don't like the implication that the focus should be on merging soft fork changes into Bitcoin Core.  IMO, the focus should be on convincing people that the changes should be made to the Bitcoin Protocol, for which Bitcoin Core is the current reference implementation.

I'm guessing your main complaint here, though, is about the word "getting", so I'm rephrasing it to "[...] of adding [...] to Bitcoin".  Is that acceptable?

Reply:
That reasoning makes a lot of sense. Change looks good!